src/screens/TopicPageScreen/index.js

```diff
@@ -53,7 +54,9 @@ const TopicPageScreen = (props) => {
 
   const [offset, setOffset] = React.useState(0);
   const [client] = React.useContext(Context).client;
+  const [user] = React.useContext(Context).profile;
   const refBlockComponent = React.useRef();
+  const bottomSheetFollowRef = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `user` context value has been destructured and a new ref `bottomSheetFollowRef` has been added. Ensure these are used appropriately in the component.
```diff
+ const [user] = React.useContext(Context).profile;
+ const bottomSheetFollowRef = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->